modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/exception/mapper/ExceptionMapper.java

```diff
@@ -32,8 +32,9 @@
 	public Response toResponse(Exception exception) {
 		Throwable throwable = exception.getCause();
 
-		javax.ws.rs.ext.ExceptionMapper exceptionMapper =
-			_providers.getExceptionMapper(throwable.getClass());
+		javax.ws.rs.ext.ExceptionMapper<Throwable> exceptionMapper =
+			_providers.getExceptionMapper(
+				(Class<Throwable>)throwable.getClass());
```

Comment:
Don't like it but it is the only way I've found to get working code and pass source formatter